src/main/java/com/techsophy/tsf/runtime/form/dto/EqualsOperation.java

```diff
@@ -0,0 +1,18 @@
+package com.techsophy.tsf.runtime.form.dto;
+
+import lombok.Data;
+import org.springframework.data.mongodb.core.query.Criteria;
+import static com.techsophy.tsf.runtime.form.constants.FormDataConstants.CONTAINS_ONLY_NUMBER;
+
+@Data
+public class EqualsOperation implements FilterOperation
+{
+    private String equals;
```

Comment:
take this as an object instead of a string, so that user can specify the exact data type.
Based on the datatype add the criteria

Reply:
> Have you missed committing files?

Actually my branch was behind latest changes so i have resolved some merge conflicts and added those after pushing

Reply:
> take this as an object instead of a string, so that user can specify the exact data type. Based on the datatype add the criteria

ok vaibhav i will specify it as object data type